src/Task/Output/Output.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * This file is part of php-block
+ *
+ * (c) Ameen Ross <a.ross@amdev.eu>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE
+ */
+
+namespace Bldr\Block\Php\Task\Output;
+
+use JakubOnderka\PhpParallelLint as Linter;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Ameen Ross <a.ross@amdev.eu>
+ */
+class Output extends Linter\TextOutput
+{
+    /**
+     * @var OutputInterface $realOutput
+     */
+    private $realOutput;
+
+    /**
+     * @param Linter\IWriter $writer
+     */
+    public function __construct(Linter\IWriter $writer)
```

Comment:
You mean just the constructor? I overrode it because in Linter\TextOutput it does redundant stuff for our usecase.